Source/AviationWeather.Test/Source/metar_parser_tests.cpp

```diff
@@ -320,13 +320,14 @@ void MetarParserTests::METAR_Parser_RunwayVisualRange()
     Assert::AreEqual(uint8_t(9), it->runway_number);
     Assert::AreEqual(uint16_t(3500), it->visibility_min);
     Assert::AreEqual(uint16_t(4500), it->visibility_max);
-    Assert::AreEqual(metar::visibility_modifier_type::none, it->visibility_modifier);
+    Assert::AreEqual(metar::visibility_modifier_type::none, it->visibility_min_modifier);
```

Comment:
Is there a reason `it->visibility_max_modifier` is checked for the others but not this oone?